src/model/specific_path/LineType.java

```diff
@@ -0,0 +1,12 @@
+package model.specific_path;
+
+public enum LineType {
+
+	Line,
+	QuadricBezier,
+	SmoothBezier,
+	Curveto,
+	SmoothCurveto,
+	Arc
+	
+}
```

Comment:
What should be the difference between Quadractic bezier and Smooth bezier? And do you think these types cover all the line types? Because if not, there will be a problem with the open/closed principle